buildSrc/subprojects/plugins/src/main/kotlin/org/gradle/plugins/performance/PerformanceTestPlugin.kt

```diff
@@ -258,7 +273,15 @@ class PerformanceTestPlugin : Plugin<Project> {
         }
 
         create("distributedPerformanceTest", DistributedPerformanceTest::class) {
-            (options as JUnitOptions).excludeCategories(performanceExperimentCategory)
+            (options as JUnitOptions).apply {
+                includeCategories(performanceRegressionTestCategory)
+                excludeCategories(slowPerformanceRegressionTestCategory)
+            }
+            channel = "commits"
+            retryFailedScenarios()
+        }
+        create("distributedSlowPerformanceTest", DistributedPerformanceTest::class) {
+            (options as JUnitOptions).includeCategories(slowPerformanceRegressionTestCategory)
             channel = "commits"
             retryFailedScenarios()
         }
```

Comment:
And `create("distributedPerformanceExperiment")` can also be removed right?

Reply:
See https://github.com/gradle/gradle/pull/10801#discussion_r326491920